### DISCUSSION TITLE
How to map "⌘ + /" to "comment line"?

### DISCUSSION THREAD

**Discussion by W-quan** [LazyVim/LazyVim#3033] [Q&A]
Like Idea.

**Comment by diocletiann** [comment]
this is how I did it:
```
vim.keymap.set("n", "<D-/>", "gccj", { remap = true })
vim.keymap.set("v", "<D-/>", "gc", { remap = true })
```

careful with 0.10, there's a UB issue with D modifier.

**Reply by andreacfromtheapp** [reply]
hi @diocletiann , does this work for `<leader>/` as well? any different way to set it in that regards? does this look good?

```lua
vim.keymap.set("n", "<leader>/", "gccj", { remap = true, desc = "Toggle comment for line" })
vim.keymap.set("v", "<leader>/", "gc", { remap = true,  desc = "Toggle comment for selection" })
```

**Reply by andreacfromtheapp** [reply]
just found out it does! thank you :)

**Reply by murlakatamenka** [reply]
Should be

```lua
{ remap = true, desc = "Toggle comment for line" }
```

**Reply by andreacfromtheapp** [reply]
absolutely. sorry about the typo! (fixing it just for better future reference.

**Reply by W-quan** [reply]
`vim.keymap.set("n", "<leader>/", "gccj", { remap = true, desc = "Toggle comment for line" }) ` not remap. still `Grep (Root Dir)`

**Comment by W-quan** [comment]
It is not working.  
nvim: v0.10.0-dev-2581+gd5488633f-Homebrew
terminal: iTerm2

**Reply by diocletiann** [reply]
works for me on Alacritty and Kitty, not sure how iTerm2 handles cmd. Also, you may hit UB using D mod on v0.10.0.

**Reply by W-quan** [reply]
What is UB and D mod?
